packages/rocketchat-livechat/server/models/LivechatInquiry.js

```diff
@@ -0,0 +1,44 @@
+class LivechatInquiry extends RocketChat.models._Base {
+	constructor() {
+		super();
+		this._initModel('livechat_inquiry');
+
+		this.tryEnsureIndex({ 'rid': 1 }); // room id corresponding to this inquiry
+		this.tryEnsureIndex({ 'name': 1 }); // name of the inquiry (client name for now)
+		this.tryEnsureIndex({ 'message': 1 }); // message sent by the client
+		this.tryEnsureIndex({ 'ts': 1 }); // timestamp
+		this.tryEnsureIndex({ 'code': 1 }); // (for routing)
+		this.tryEnsureIndex({ 'agents': 1}); // Id's of the agents who can see the inquiry (handle departments)
+		this.tryEnsureIndex({ 'status': 1}); // 'open', 'taken'
+	}
+
+	/*
+	 * mark the inquiry as taken
+	 */
+	takeInquiry(inquiryId) {
+		this.update({
+			"_id": inquiryId
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2727001283/issues/source?bid=3199634&fileBranchId=3348698#l20)